fpsd/features.py

```diff
@@ -0,0 +1,765 @@
+#!/usr/bin/env python3.5
+import pandas as pd
+import os
+from sqlalchemy import create_engine
+from tqdm import tqdm
+import subprocess
+
+
+def compute_bursts(df):
+    """Compute list of burst lengths for a given trace
+    """
+    bursts = []
+    current_burst_length = 0
+
+    for row in df.itertuples():
+        if row.ingoing == False:
+            current_burst_length += 1
+        elif row.ingoing == True and current_burst_length > 0:
```

Comment:
I agree - I’ve rewritten `compute_bursts()` to compute both. I've also modified the corresponding tests to reflect the new functionality (e.g. see in `test.test_features.RawFeatureGenerationTest.test_burst_table_creation`)